core/templates/dev/head/library/library.html

```diff
@@ -56,16 +56,13 @@ <h2 class="oppia-library-h2" translate="I18N_LIBRARY_SUB_HEADER"></h2>
                translate="I18N_LIBRARY_NO_EXPLORATION_GROUPS">
           </div>
 
-          <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index">
+          <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index" ng-init="groupIndex = $index">
             <div class="row oppia-library-group-header-container">
               <h2 class="oppia-library-group-header">
                 <a ng-click="showAllResultsForCategories(group.categories)" translate="<[group.translationId]>">
                 </a>
+                <span ng-show="currentGroupIndex == $index"> &gt; </span>
```

Comment:
Also, can you try using this instead? https://design.google.com/icons/#ic_keyboard_arrow_right

No need to download a png image or anything -- have a look at how we use material icons elsewhere in the codebase. I think you just use `<i class="material-icons">&#xE315;</i>` and add size classes if needed.